.gitignore

```diff
@@ -1,3 +1,7 @@
+# OS, IDEs, tooling
+.idea/
+.python-version
+
```

Comment:
Would generally agree that very user-specific configs are better handled by users themselves, but I'd put files which originate from very popular dev tools or often used language-related tooling in the same bucket as common system files.

 `.idea` is where PyCharm saves all project data, which is equivalent to what I believe `.vscode` is in VSCode, and AFAIK those are the two most widespread IDEs for Python development. `.python-version` is a file used by pyenv, which is a popular Python version manager.

... I personally always gitignore e.g. `.DS_Store` files in all projects even though I don't develop on Mac. It's more meant as courtesy to others + to help reduce overhead (messy PRs) because I don't assume everyone knows how to use global ignore rules (+ there are people who `git add .` by default :woozy_face:). Same for filtering for `tmp`, `bakk` etc. In the case of Python/Django projects, I also think it's a good idea to always ignore e.g. `.env*` files, just so no-one accidentally shares potentially sensitive data. ... So, normally, I'd have added a bunch more rules but wanted to keep this low-key, lol.